### DISCUSSION TITLE
Running CWatM calibration on Ubuntu requires modification to set file permissions

### DISCUSSION THREAD

**Discussion by abbylute** [iiasa/CWatM#79] [General]
I am running a trial model calibration following the 'What is Needed' section here: [https://cwatm.iiasa.ac.at/calibration.html]. I am running on a google compute virtual machine with Ubuntu 20.04.6 LTS.

I found that the model failed to run because the runp*_*.sh files it created did not have execute permission. I was able to fix this by adding `os.chmod(runfile, 0o744)` on line 217 of calibration_single.py. I also had the add `os.chmod(runfile, 0o744)` on line 539 (now line 540 after the first edit) to get the 'best' model to run.

I was surprised to need to modify the code in order to get this to run. Perhaps there is a better way? Or maybe I'm doing something incorrectly? If not, perhaps these lines should be added to the hosted code.

**Comment by mikhailsmilovic** [comment]
This is great, danke schön! 
This has now been included in the code: https://github.com/iiasa/CWatM/commit/31ed493131202c16483a02a7192411fc78d76342

**Reply by abbylute** [reply]
Great, thanks!
